frontend/src/pages/org/index.ts

```diff
@@ -256,15 +256,15 @@ export class Archive extends LiteElement {
         <div class="border-b" role="rowgroup">
           <div class="flex font-medium" role="row">
             <div class="w-1/2 px-3 py-2" role="columnheader" aria-sort="none">
-              ${msg("Name", { desc: "Team member's name" })}
+              ${msg("Name", { desc: "Team member's name", id: "teammembername" })}
```

Comment:
@SuaYoo added a manual id here for this string because `lit-localize` was complaining about having multiple "Name" strings with different descriptions

Reply:
Done!